cmd/flags_misc.go

```diff
@@ -31,6 +31,7 @@ var (
 	metricsFlag         = "metrics"
 	metricsEndpointFlag = "metrics.endpoint"
 	metricsTlS          = "metrics.tls"
+	libp2pMetrics       = "libp2p.metrics"
```

Comment:
optional:

As we have module namespaces, e.g. p2p/header/share/etc, it would be nice just to keep it this way, so instead of `libp2p.metrics` it is just `p2p.metrics`

Reply:
OK, makes sense

BUT: don't you think that this would be confusing when we add `BandwidthCounter` and Stats metrics? _We can cross that bridge when we arrive at it, just flagging for the moment._

Reply:
I wonder if we really need bandwitdthCounter if enable native libp2p metrics?

Reply:
Good question 🤔 will investigate and let you know